From 248319c79ae529ee3f95f59793b2e3d58b9d8f87 Mon Sep 17 00:00:00 2001
From: Nikolay Aleksandrov <nikolay@redhat.com>
Date: Fri, 5 Sep 2014 11:03:47 -0400
Subject: [ethernet] sfc: Correct comment about number of TX queues used on EF10

Message-id: <1409915048-26438-12-git-send-email-nikolay@redhat.com>
Patchwork-id: 91414
O-Subject: [RHEL7.1 PATCH BZ 1110888 11/32] sfc: Correct comment about number of TX queues used on EF10
Bugzilla: 1110888
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Ben Hutchings <bhutchings@solarflare.com>

EF10 implements option descriptors to switch TX checksum offload
on and off between packets.  We could therefore use a single
hardware TX queue per kernel TX queue, although we don't yet.

Signed-off-by: Ben Hutchings <bhutchings@solarflare.com>
Signed-off-by: Shradha Shah <sshah@solarflare.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit aa3930ee8cc7ab56f8a62d4e80e178ab520dca87)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/sfc/ef10.c b/drivers/net/ethernet/sfc/ef10.c
index 174a92f..3b39798 100644
--- a/drivers/net/ethernet/sfc/ef10.c
+++ b/drivers/net/ethernet/sfc/ef10.c
@@ -172,8 +172,8 @@ static int efx_ef10_probe(struct efx_nic *efx)
  struct efx_ef10_nic_data *nic_data;
  int i, rc;
 
- /* We can have one VI for each 8K region.  However we need
-  * multiple TX queues per channel.
+ /* We can have one VI for each 8K region.  However, until we
+  * use TX option descriptors we need two TX queues per channel.
   */
  efx->max_channels =
   min_t(unsigned int,
-- 
1.7.1